src/femr/featurizers/featurizers.py

```diff
@@ -220,8 +227,20 @@ def __init__(
                 self.time_bins
             ), f"You cannot have duplicate values in the `time_bins` argument. You passed in: {self.time_bins}"
 
+    def code_is_excluded(self, code: int) -> bool:
+        """Check if `code` satisfies `code_criteria`."""
+        if isinstance(self.excluded_codes, set):
```

Comment:
Please move this type check to the constructor so we can give errors much earlier in the pipeline